llvm/test/tools/sycl-post-link/spec-constants/multiple-sym-id-usages.ll

```diff
@@ -1,16 +1,14 @@
 ; This test checks that the tool does not crash and removes the unused spec
 ; constant global symbol when it is referenced more than once.
```

Comment:
```suggestion
; This test checks that the tool does not crash when spec constant
; global symbol is referenced more than once.
```

@mlychkov, does this change look good to you?

Reply:
Yes, thank you.